utils/dataset_manifest/README.md

```diff
@@ -0,0 +1,71 @@
+## Simple command line to prepare dataset manifest file
+
+### Steps before use
+
+When used separately from Computer Vision Annotation Tool(CVAT), the required modules must be installed
+
+```bash
+python3 -m venv .env
+. .env/bin/activate
+pip install -U pip
+pip install -r requirements.txt
+```
+
+### Using
+
+```bash
+usage: python create.py [-h] --type {video,images} [--chunk_size CHUNK_SIZE] manifest_directory sources [sources ...]
+
+positional arguments:
+  manifest_directory    Directory where the manifest file will be saved
+  sources               Source paths
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --type {video,images}
```

Comment:
@Marishka17 , When is it critical to have `--type` argument? Can we detect it automatically?

Reply:
Not critical, I think yes. Will be fixed